.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-Daether.checksums.algorithms=SHA-512,SHA-256,SHA-1,MD5
+-Daether.connector.smartChecksums=false
```

Comment:
I don't know what aether is. Is that for the mvnw downloader/verification or something else?

Reply:
For reference, this is aether -> https://github.com/apache/maven-resolver.